Website/src/items/content/EdgeWebsite/home/speakers/Page Components/speakers-jss-main-FeaturedSpeakers-3.yml

```diff
@@ -0,0 +1,41 @@
+﻿---
+ID: "be567aca-7ee4-5063-b7f1-29ead100ba3a"
+Parent: "f0c77235-5000-532c-a61c-e5f11947760d"
+Template: "fdf43aad-c5d1-54e4-9909-59bfddc86689"
+Path: "/sitecore/content/EdgeWebsite/home/speakers/Page Components/speakers-jss-main-FeaturedSpeakers-3"
+Languages:
+- Language: en
+  Fields:
+  - ID: "b5e02ad9-d56f-4c41-a065-a133db87bdeb"
+    Hint: __Display name
+    Value: "FeaturedSpeakers-3"
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "097edbee-00fc-5fe5-ae17-e4cfb6c43d09"
+      Hint: Title
+      Value: Featured Speakers
+    - ID: "25bed78c-4957-4165-998a-ca1b52f67497"
+      Hint: __Created
+      Value: 20210714T203228Z
+    - ID: "83401b48-7ea2-5365-bb09-0e4a75ca7b06"
+      Hint: Subtitle
+      Value: "Road-test the world's most trusted sports and fitness equipment – we'll be welcoming 2,000 brands at this year's PLAY! Summit."
```

Comment:
This text does not really fit a "Featured speakers" section. Could it be changed?